eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringMongoReader.java

```diff
@@ -58,13 +59,16 @@
 
     private int chunkSize;
 
+    private boolean isClusteredVariantsMongoReader;
+
     public ClusteringMongoReader(MongoClient mongoClient, String database, MongoTemplate mongoTemplate,
-                                 String assembly, int chunkSize) {
+                                 String assembly, int chunkSize, boolean isClusteredVariantsMongoReader) {
         this.mongoClient = mongoClient;
         this.database = database;
         this.mongoTemplate = mongoTemplate;
         this.assembly = assembly;
         this.chunkSize = chunkSize;
+        this.isClusteredVariantsMongoReader = isClusteredVariantsMongoReader;
```

Comment:
Since this particular variable name isn't self-explanatory, suggest adding a comment as to why we need this.

Reply:
changed the name to **readOnlyClusteredVariants** to make it more clear and also added comment